slack.coffee

```diff
@@ -9,9 +9,18 @@ exports.use = (robot) ->
     # reacts to only messages by insalled workspace users in a shared channel
     installedTeamOnly: process.env.INSTALLED_TEAM_ONLY?
   try
-    options.rtm = JSON.parse(process.env.HUBOT_SLACK_RTM_CLIENT_OPTS)
-  catch
+    options.rtm = JSON.parse(process.env.HUBOT_SLACK_RTM_CLIENT_OPTS or '{}')
```

Comment:
If you set the env variable to an empty string, the parsing fails. This change improves the behavior in the scenario.